atest/testdata/running/continue_on_failure_tag.robot

```diff
@@ -0,0 +1,265 @@
+*** Variables ***
+${HEADER}                 Several failures occurred:
+
+*** Test Cases ***
+Continue in test with tag
+    [Documentation]    FAIL ${HEADER}\n\n
+    ...    1) 1\n\n
+    ...    2) 2
+    [Tags]   robot:continue-on-failure
+    Fail   1
+    Fail   2
+    Log    This should be executed
+
+Continue in test with set tag
+    [Documentation]    FAIL ${HEADER}\n\n
+    ...    1) 1\n\n
+    ...    2) 2
+    Set Tags   robot:continue-on-failure
+    Fail   1
+    Fail   2
+    Log    This should be executed
+
+Continue in user kewyord with tag
+    [Documentation]    FAIL ${HEADER}\n\n
+    ...    1) kw1a\n\n
+    ...    2) kw1b
+    Failure in user keyword with tag
+    Fail   This should not be executed
+
+Continue in test with tag and UK without tag
+    [Documentation]    FAIL ${HEADER}\n\n
+    ...    1) kw2a\n\n
+    ...    2) This should be executed
+    [Tags]   robot:continue-on-failure
+    Failure in user keyword without tag
+    Fail   This should be executed
+
+Continue in test with tag and nested UK with and without tag
+    [Documentation]    FAIL ${HEADER}\n\n
+    ...    1) kw1a\n\n
+    ...    2) kw1b\n\n
+    ...    3) kw2a\n\n
+    ...    4) This should be executed
+    [Tags]   robot:continue-on-failure
+    Failure in user keyword with tag     run_kw=Failure in user keyword without tag
+    Fail   This should be executed
+
+Continue in test with tag and two nested UK with tag
+    [Documentation]    FAIL ${HEADER}\n\n
+    ...    1) kw1a\n\n
+    ...    2) kw1b\n\n
+    ...    3) kw1a\n\n
+    ...    4) kw1b\n\n
+    ...    5) This should be executed
+    [Tags]   robot:continue-on-failure
+    Failure in user keyword with tag     run_kw=Failure in user keyword with tag
+    Fail   This should be executed
+
+Continue in for loop with tag
+    [Documentation]    FAIL ${HEADER}\n\n
+    ...    1) loop-1\n\n
+    ...    2) loop-2\n\n
+    ...    3) loop-3
+    [Tags]   robot:continue-on-failure
+    FOR    ${val}    IN    1    2    3
+        Fail   loop-${val}
+    END
+
+Continue in for loop with set tag
+    [Documentation]    FAIL ${HEADER}\n\n
+    ...    1) loop-1\n\n
+    ...    2) loop-2\n\n
+    ...    3) loop-3
+    FOR    ${val}    IN    1    2    3
+        Set Tags   robot:continue-on-failure
+        Fail   loop-${val}
+    END
+
+Continue in for loop without tag
+    [Documentation]    FAIL loop-1
+    FOR    ${val}    IN    1    2    3
+        Fail   loop-${val}
+    END
```

Comment:
This seems to be testing the normal behavior that ought to be tested thoroughly elsewhere. Perhaps having an example here is fine, but at least the name should be changed to something like "No continue ...".